WcaOnRails/app/assets/stylesheets/users.scss

```diff
@@ -19,3 +19,16 @@ input[type="radio"].center-vertically {
   vertical-align: middle;
   height: 100%;
 }
+
+.input-group.date.datetimepicker {
+  width: 100%;
+}
+
+.form-group {
```

Comment:
Maybe it should, but it won't.
As you commented before, the field was smaller and had square borders. That's because it is a single child inside a div, and the cool looking styling requires 2 childs (the calendar button and the input) and does some magic.
We could try to dig into bootstrap and figure this out and override, but I don't think it would look much prettier than this...